lib/ExtraCorres.thy

```diff
@@ -8,6 +8,15 @@ theory ExtraCorres
 imports Corres_UL
 begin
 
+(* FIXME: the S in this rule is mainly to make the induction work, we don't actually need it in
```

Comment:
Not sure why this and the comments below are FIXMEs rather than a normal comment? The contents of the comment is fine.

Reply:
Should probably split it up into two comments. One part is FIXME and should be fixed, the other part is just warning for future me not to prove the strong version again to realise again that it doesn't help and is not actually stronger.